### PR TITLE
DGS-181 back end fucntiuonality added for preparation to expose new image for carrier frontend"

### PR DESCRIPTION
**Pull request by arturas-bes**
… carrier frontend

### CODE REVIEW

**Comment by ErnestasLuza** [comment]
Front-end added

File: src/Repository/ProductRepository.php [hunk 1]
```diff
@@ -31,6 +31,20 @@ public function getAllActiveProducts($onlyId = false)
         return $this->db->getValue($query);
     }
 
+    public function getAllActiveDpdProductReferences($onlyId = false)
```
Comment (tomjas1997) [inline src/Repository/ProductRepository.php:34]: Might as well create two different function, one for product information one for only id_reference.

File: views/js/front/sustainable.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+$(document).ready(function () {
```
Comment (tomjas1997) [inline views/js/front/sustainable.js:1]: Naming of file is quite weird, sustainableCarriers maybe?

File: controllers/admin/AdminDPDBalticsAjaxController.php [hunk 3]
```diff
@@ -51,4 +51,10 @@ public function ajaxProcessImportParcels()
         $countryIso = Country::getIsoById($countryId);
         $this->ajaxDie(json_encode($parcelShopImport->importParcelShops($countryIso)));
     }
+
+    public function ajaxProcessMyAction()
+    {
+       Tools::getValue('dataPassed');
```
Comment (tomjas1997) [inline controllers/admin/AdminDPDBalticsAjaxController.php:57]: what is this code? It does nothing.

File: src/Repository/ProductRepository.php (hunk 1 above)
Comment (tomjas1997) [reply src/Repository/ProductRepository.php:34]: Seems like a new function, could still split it into two actually usable functions with SOLID principles

**Review by tomjas1997 — APPROVED** [review]
Still that one function bothers me, but i guess it works so approving.

**Pull request merged**